plugins/backend/catalog-backend-module-okta/src/providers/OktaOrgEntityProvider.ts

```diff
@@ -127,15 +165,26 @@ export class OktaOrgEntityProvider extends OktaEntityProvider {
                 this.logger.warn(`failed to add user to group: ${e.message}`);
               }
             });
+            let parentGroup: Group | undefined = undefined;
```

Comment:
Looks like these 20 lines are duplicated so could be extracted out

Reply:
done.